src/app/banner/banner.component.html

```diff
@@ -1,22 +1,13 @@
-<div class="identity-banner w-100" *ngIf="!globalVars.webview">
-  <div
-    class="d-flex justify-content-between h-100 align-items-center banner-margin"
-  >
-    <img src="/assets/camelcase_logo.svg" class="home-logo" />
-    <div
-      class="fs-18px font-weight-semibold d-flex align-items-center verify-site-banner"
-    >
-      <span class="fc-title">Always verify you are on: </span>
-      <div>
-        <img
-          class="ml-15px position-relative"
-          src="/assets/lock-green.svg"
-          style="top: -2px"
-        />
-        <span class="fc-green ml-5px">{{
-          globalVars.environment.hostname
-        }}</span>
-      </div>
+<div class="section--header" *ngIf="!globalVars.webview">
```

Comment:
Banner is looking a bit bunched up on mobile: 
![image](https://github.com/deso-protocol/identity/assets/81658138/5bf12a15-b9b5-428d-a98b-52d269614936)

Reply:
Actually everything is a bit wonky on mobile:
![image](https://github.com/deso-protocol/identity/assets/81658138/b2377211-7b6a-4a40-92c1-56587098a6f7)

Reply:
Yup, will do a mobile pass.